test/e2e/platform_wan_test.go

```diff
@@ -455,18 +456,177 @@ var _ = Describe("Hazelcast WAN", Label("platform_wan"), func() {
 	})
 
 	Context("Data Sync between separate clusters", func() {
-		It("shouldn't fail in active-passive mode across separate clusters", Serial, Tag(EE|AnyCloud), func() {
-			if !ee {
-				Skip("This test will only run in EE configuration")
+		It("should replicate data for maps with TS and non TS storage in active-passive WAN replication mode", Serial, Tag(EE|AnyCloud), func() {
+			SwitchContext(context1)
+			setupEnv()
+			setLabelAndCRName("hpts-1")
```

Comment:
```suggestion
			setLabelAndCRName("hpwts-1")
```
We are already using `hpts` in the platform tiered storage suite.